ArduCopter/Copter.h

```diff
@@ -472,6 +472,7 @@ class Copter : public AP_HAL::HAL::Callbacks {
     // Current location of the copter (altitude is relative to home)
     Location_Class current_loc;
     Location ekf_origin;
+    Vector3f current_pos;
```

Comment:
The problem with those data is that their aren't exactly what is in EKF and need to allocate either a structur or a float vect to be get ... Plus by updating them only once we reach the inertia callback, it garanties that the behaviour remain the same than with inertialNav that was making a copy too.
But I am not blocked on that!